README.md

```diff
@@ -32,6 +32,15 @@ You can also track output of datagrid nodes calling:
 After calling either of `datagrid/deploy.sh` or `datagrid/redeploy.sh`, call `datagrid/test.sh` to quickly test that the data grid responds. 
 
 
+# Delayed train dashboard
+
+As a stepping stone for workshop attendees, a Java FX dashboard of delayed trains has been developed.
```

Comment:
Could you file an issue so that we switch to a web view in the future? Patternfly has a list view component with a nice rendering.

Reply:
#11